Mage.Sets/src/mage/cards/s/StrongholdGambit.java

```diff
@@ -63,21 +63,21 @@ public boolean apply(Game game, Ability source) {
         Player controller = game.getPlayer(source.getControllerId());
         MageObject sourceObject = game.getObject(source);
         if (controller != null && sourceObject != null) {
-            Map<UUID, UUID> choosenCard = new LinkedHashMap<>();
+            Map<UUID, UUID> chosenCard = new LinkedHashMap<>();
```

Comment:
While your fixing spelling, these should really be chosenCards since it holds multiple